x/launch/types/message_request_add_account_test.go

```diff
@@ -0,0 +1,58 @@
+package types_test
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func TestMsgRequestAddAccount_ValidateBasic(t *testing.T) {
+	chainID, _ := sample.ChainID(10)
+	tests := []struct {
+		name string
+		msg  types.MsgRequestAddAccount
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: types.MsgRequestAddAccount{
+				Address: "invalid_address",
+			},
+			err: sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress,
+				"invalid address (invalid_address): decoding bech32 failed: invalid index of 1"),
+		}, {
+			name: "invalid chain id",
+			msg: types.MsgRequestAddAccount{
+				Address: sample.AccAddress(),
+				ChainID: "invalid_chain",
+			},
+			err: sdkerrors.Wrapf(types.ErrInvalidChainID, "invalid_chain"),
+		}, {
+			name: "invalid chain name",
```

Comment:
We don't have chain name for this message, what's the difference with the test above?

Reply:
the chainID format is right, but the name is wrong because we have an invalid character (`"wh.thc-10"`)